tools/install_cluster_systemd/README.md

```diff
@@ -0,0 +1,2 @@
+####
```

Comment:
I think it would belong more to: 

https://github.com/etcd-io/etcd/tree/main/contrib/ansible. 

Please notice that we have already: 
`https://github.com/etcd-io/etcd/tree/main/contrib/systemd`